2023-04-08 19:11:18 PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
2023-04-08 19:11:18 NAME="Debian GNU/Linux"
2023-04-08 19:11:18 VERSION_ID="11"
2023-04-08 19:11:18 VERSION="11 (bullseye)"
2023-04-08 19:11:18 VERSION_CODENAME=bullseye
2023-04-08 19:11:18 ID=debian
2023-04-08 19:11:18 HOME_URL="https://www.debian.org/"
2023-04-08 19:11:18 SUPPORT_URL="https://www.debian.org/support"
2023-04-08 19:11:18 BUG_REPORT_URL="https://bugs.debian.org/"

File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: libGL.so.1: cannot open shared object file: No such file or directory